nix/pkgs/android-sdk/default.nix

```diff
@@ -3,7 +3,7 @@
 # for the Android development environment.
 #
 
-{ callPackage }:
+{ callPackage, lib, stdenv }:
```

Comment:
Why are you adding this here?

Reply:
I though it's needed to pass through the arguments.
But it's not. Probably some callPackage magic, will read about it.